.travis.yml

```diff
@@ -1,23 +1,24 @@
-sudo: false
 language: php
+
+php:
+    - 5.4
+    - 5.5
+    - 5.6
+    - 7.0
+    - 7.1
+    - 7.2
+
 matrix:
-  fast_finish: true
   include:
-    - php: 5.4
-    - php: 5.5
-    - php: 5.6
-    - php: 7.0
-    - php: 7.1
-    - php: hhvm
-      sudo: true
-      dist: trusty
-      group: edge # until the next Travis CI update
-  allow_failures:
-    - php: hhvm
-    - php: 7.0
-    - php: 7.1
+    - php: 5.3
```

Comment:
Again, we should skip this version because it is dead.

Reply:
I will remove these PHP versions.